sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -3144,6 +3148,10 @@ SIREPO.app.directive('simStatusPanel', function(appState) {
                 return callSimState('getAlert');
             };
 
+            $scope.cancelledDueToTimeout = function() {
```

Comment:
It's ok to have this as the name, but then the cancelledAfterSecs (probably formatted as a human readable time string, e.g. 5 minutes or 1 hour) has to be available to the template.

Reply:
I've mad cancelledAfterSecs available and I'm doing some conversion to format the time. Look ok?

https://github.com/radiasoft/sirepo/pull/2373/files#diff-138a2b807a59f609d46eb468d799eaa1R317